Client/Frontend/Login Management/BreachAlertsManager.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  BreachAlertsManager.swift
+//  Client
+//
+//  Created by Vanna Phong on 5/21/20.
+//  Copyright © 2020 Mozilla. All rights reserved.
+//
+
+import Foundation
+import Storage // or whichever module has the LoginsRecord class
+import Shared // or whichever module has the Maybe class
+
+/// Breach structure decoded from JSON
+struct BreachRecord: Codable {
+    var name: String
+    var title: String
+    var domain: String
+    var breachDate: String
+    var description: String
+
+    enum CodingKeys: String, CodingKey {
+        case name = "Name"
+        case title = "Title"
+        case domain = "Domain"
+        case breachDate = "BreachDate"
+        case description = "Description"
+    }
+}
+
+/// A manager for the user's breached login information, if any.
+final public class BreachAlertsManager {
+    var breaches: [BreachRecord] = []
+    var breachAlertsClient: BreachAlertsClientProtocol
+
+    init(_ client: BreachAlertsClientProtocol = BreachAlertsClient()) {
+        self.breachAlertsClient = client
+    }
+
+    /// Loads breaches from Monitor endpoint using BreachAlertsClient.
+    ///    - Parameters:
+    ///         - completion: a completion handler for the processed breaches
+    func loadBreaches(completion: @escaping (Maybe<[BreachRecord]>) -> Void) {
+        print("loadBreaches(): called")
+
+        DispatchQueue.global(qos: .background).async {
+            self.breachAlertsClient.fetchData(endpoint: .breachedAccounts) { maybeData in
```

Comment:
You don't need to manually set this to a background thread, since it has a completion handler and data tasks run asynchronously, it will already be on a background asynchronous thread.